megamek/src/megamek/common/Entity.java

```diff
@@ -9347,6 +9347,16 @@ public boolean shouldDeploy(int round) {
         return (!deployed && (getDeployRound() <= round) && !isOffBoard());
     }
 
+    /**
+     * Returns true if the offboard entity should be deployed this round.
+     * @param round The current round number.
+     * @return True if and only if the offboard entity should deploy this
+     *         round, otherwise false.
+     */
+    public boolean shouldOffboardDeploy(int round) {
+        return (isOffBoard() && (!deployed && (getDeployRound() <= round)));
```

Comment:
A little too many parens here.

Reply:
I used the same number as the `shouldDeploy` routine (my last PR had too few; so I guess I'm binary searching).

Reply:
:)